src/store/modules/wallet.js

```diff
@@ -199,6 +222,10 @@ const getters = {
       return count;
     }, 0);
   },
+  getTotalSales: state => state.totalSales,
+  getTotalRoyalty: state => state.totalRoyalty,
+  getSalesDetails: state => state.salesDetails,
+  getRoyaltyDetails: state => state.royaltyDetails,
```

Comment:
Prefer name not starting with `get` as it is not a function
```suggestion
  walletTotalSales: state => state.totalSales,
  walletTotalRoyalty: state => state.totalRoyalty,
  walletSalesDetails: state => state.salesDetails,
  walletRoyaltyDetails: state => state.royaltyDetails,
```